src/app/app.component.ts

```diff
@@ -45,21 +46,35 @@ export class ConferenceApp {
     { title: 'Login', component: LoginPage, icon: 'log-in' },
     { title: 'Signup', component: SignupPage, icon: 'person-add' }
   ];
-  rootPage: any = TutorialPage;
+  rootPage: any;
 
   constructor(
     public events: Events,
     public userData: UserData,
     public menu: MenuController,
-    platform: Platform,
-    confData: ConferenceData
+    public platform: Platform,
+    public confData: ConferenceData,
+    public storage: Storage,
+    public toastCtrl: ToastController
   ) {
     // Call any initial plugins when ready
     platform.ready().then(() => {
       StatusBar.styleDefault();
       Splashscreen.hide();
     });
 
+    // Check if the user has already seen the tutorial
+    this.userData.checkHasSeenTutorial().then((hasSeenTutorial) => {
+      if (hasSeenTutorial === null) {
+        // User has not seen tutorial
+        this.rootPage = TutorialPage;
+        this.storage.set('hasSeenTutorial', 'true');
```

Comment:
I don't think we should set this to `true` until the user presses "Skip" or "Continue" from the tutorial. If I launch the app on my phone and then exit the app without interacting with it the next time I open it the tutorial will be skipped. Then I have to open the menu to see it again when I never really _said_ I saw it.